.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,20 @@
+name: ❓ Question
```

Comment:
Are questions encouraged in this repo? Maybe I misunderstood #534, but maybe the Winterbloom Discord server (under the `#nox` channel) could fit better for questions.

However, we could keep this template, and also provide a link in a `ISSUE_TEMPLATE/config.yml` [^1] file for those who want to ask on DIscord.

Just tagging @wntrblm/nox to attract the maintainers' attention, to get some more opinions.

[^1]: `config.yml` reference is [here at the GitHub docs](https://docs.github.com/es/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

Reply:
I like that idea!

Reply:
I think I can write a `config.yml` after this PR, unless someone disagrees :)